report/styled_par.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * This example showcases the creation of styled paragraph
+ * The output is saved as styled_paragraph.pdf which illustrates some of the features
+ * of the creator.
+ */
+/*
+ * NOTE: This example depends on github.com/unidoc/unipdf/v3/creator, MIT licensed,
+ *       and github.com/unidoc/unipdf/v3/model,
```

Comment:
This is incorrect.  unipdf is AGPL licensed with a commercial clause.  No need to include that here. Please keep consistent with other examples here.

Reply:
Done